apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/BlockchainProcessorImpl.java

```diff
@@ -102,6 +103,7 @@
     // TODO: YL remove static instance later
    private static final PropertiesHolder propertiesHolder = CDI.current().select(PropertiesHolder.class).get();
    private static final BlockchainConfig blockchainConfig = CDI.current().select(BlockchainConfig.class).get();
+   private static final DexService dexService = CDI.current().select(DexService.class).get();
```

Comment:
I suppose, **static** is not needed here, even PropertiesHolder, BlockchainConfig can be NON static